src/metabase/pulse/parameters.clj

```diff
@@ -15,19 +15,24 @@
   [_pulse dashboard]
   (:parameters dashboard))
 
+(defn param-val
+  "Returns the parameter value, falling back to :default if :value is not present."
+  [parameter]
+  (get parameter :value (:default parameter)))
```

Comment:
This is fine. Sometimes we just use `(some-fn :value :default)` to define such a function.

Reply:
another option could be `(or (:value parameter) (:default parameter)`

Reply:
Both of these suggestions fail in the case when we want a nil `:value` to override the default.  It should only fallback to the default if the `:value` key is not present.

Reply:
👍 fair, didn't realize that distinction. Maybe it would be worth making it more explicit in the doc string so that nobody accidentally breaks that? (the current wording is accurate, but doesn't make it super obvious that this case is important)

Reply:
Thanks, is this alright? 7070d65fd5491300c84f7242d382c708213b8c8f

Reply:
Looks good!